### DISCUSSION TITLE
Is it possible to create a countdown field?

### DISCUSSION THREAD

**Discussion by uprog89** [directus/directus#2727] [Archive]
date a start and date end = count day left (diffday)

**Comment by benhaynes** [comment]
Yeah — seems pretty easy! I assume this would be:

* A `datetime` or `date` interface that saves the future date you're counting down to
* The interface itself would just find the diff and display it formatted:
    * "Units" would adjust based on how far away the date is. So if it's months away it would use months, days, hours... if it's soon it would use hours, minutes, seconds. This could also have a color change to show when it gets close.
* If this is _readonly_, you can stop there. If you also want to manage the time in the App then you'd want to add a button to toggle between _countdown_ and _input_.

### Design Mockup:
  
![Screen Shot 2020-05-27 at 9 52 48 AM](https://user-images.githubusercontent.com/522079/83028827-f12daa00-9fff-11ea-9b37-65b8a85be563.png)

**Reply by uprog89** [reply]
Hey. I understand that you need to create your own interface for the field. I added it according to the documentation, but the field is not saved when adding it.

**Reply by uprog89** [reply]
It turned out to make days left and days left. Tell me how to change the status color? If after that red
